oss/interval_tree/IntervalTree.h

```diff
@@ -22,6 +22,22 @@ namespace interval_tree
             start(std::min(s, e)), stop(std::max(s, e)), value(v)
         {
         }
+
+        Interval()

```

Comment:
I have made a PR to the original repo with these changes as well, but since that project is very slow moving I believe we're going with a 'lets also do the changes locally for now while waiting for them to get ingested'